dev/documentation/en/datamodel-keyconcepts.md

```diff
@@ -0,0 +1,100 @@
+---
+book: dev-documentation
+version: 3.4
+title: Entities Key Concepts - Technical Documentation - OJS|OMP|OPS
+---
+
+# Entities: Key Concepts
```

Comment:
I wonder if it makes sense to call this the Data Model page and move it up after Getting Started. For example, the Architecture page includes a section on [contexts](https://docs.pkp.sfu.ca/dev/documentation/en/architecture#contexts) which could be pulled out of that and added to this page.